query-service-impl/src/test/resources/application.conf

```diff
@@ -42,6 +44,9 @@ service.config = {
       type = pinot
       clientConfig = zookeeper
       requestHandlerInfo = {
+        tenantColumnName = tenant_id
+        slowQueryThresholdMs = 100
```

Comment:
Why is this being configured to a request handler? Isn't a slow request a slow request regardless of where it's being executed?

Reply:
Flexibility. Applying the same slow query threshold for all tables and data stores isn't ideal and would give lot of false positives. Since each view/table could be different and query patterns on them could be different, this would allow that flexibility.
There is a default anyways that's applicable for all and it's not mandatory to override. In fact, i'm not setting this at handler level for actual service. This is a test.